pyproject.toml

```diff
@@ -30,7 +27,8 @@ select = [
     "W",   # pycodestyle warning
 ]
 line-length = 110
-target-version = "py38"
+target-version = "py39"
+ignore = ['COM812', 'ISC001']
```

Comment:
Just to understand: why ignore the two and do that in this pull request? I ran ruff locally with the new target version and it had no complaints without the two. Should this be dropped? Would like to understand.

Reply:
I reverted this change, but when I run `ruff format .`, I see this warning:

```
warning: The following rules may cause conflicts when used with the formatter: `COM812`, `ISC001`. To avoid unexpected behavior, we recommend disabling these rules, either by removing them from the `select` or `extend-select` configuration, or adding them to the `ignore` configuration.
```

Reply:
@jairhenrique thanks for the explanation! I have no strong opinion about it — please create a dedicated pull request if you are pro this addition e.g. if you see the warning repeatedly.